polling_stations/settings/constants/directions.py

```diff
@@ -4,5 +4,10 @@
 GOOGLE_API_KEYS = []
 BASE_GOOGLE_URL = "https://maps.googleapis.com/maps/api/directions/json?units=imperial"
 
+# Mapbox settings used by directions helper
+MAPBOX_API_KEY = ""
```

Comment:
Couple of quick comments here. The first is blindingly obvious - remember to do a corresponding PR to the polling_deploy repo which adds this setting to production.py and populates it from the vault :) Less obviously, one of the things we did in WDIV is we added some [django system checks](https://docs.djangoproject.com/en/3.2/topics/checks/) to try and help guide setup and/or prevent ourselves from misconfiging the app. There are a few of these knocking about but the system checks for the data finder app are in https://github.com/DemocracyClub/UK-Polling-Stations/blob/c5c9c1cd4290bbb0bc67d3552cafbd98c0aae60e/polling_stations/apps/data_finder/checks.py Tbh, there is some cruft here. I don't think we have used MapQuest in looooong time and I deleted all the code for it, but maybe this is a good opportunity to update these with the current constraints/assumptions.

Reply:
Blindingly obvious, often isn't! In this case I'd had a go - https://github.com/DemocracyClub/polling_deploy/pull/266.

Good pointer about the checks - that's a module (and concept - django checks) I had no awareness of even existing :smile:

Reply:
Going to punt this to here https://github.com/DemocracyClub/UK-Polling-Stations/issues/3751